src/GUI/RevEng.Core/README.md

```diff
@@ -0,0 +1,45 @@
+You need to do some setup for your app after the `Reverse Engineer` created the Entity Model classes for you. Here are some hints:
+
+## ASP.NET Core 6.0
+
+1. Register your data context class to `IServiceCollection` in your `Program.cs` file.
+
+    ```cs
+    builder.Services.AddDbContext<[ContextName]>(options => {
+        options.UseSqlServer(builder.Configuration.GetConnectionString("DefaultConnection"));

```

Comment:
This will be provider dependent

Reply:
Yes, you're right. So I plan to put `[ProviderName]` placeholder like this.

```cs
options.Use[ProviderName](builder.Configuration.GetConnectionString("DefaultConnection"));
```